packages/app/src/domain/variants/variants-table-tile/index.tsx

```diff
@@ -42,17 +44,22 @@ export function VariantsTableTile({
 
       {text.varianten_tabel.belangrijk_bericht && (
         <WarningTile
-          message={text.varianten_tabel.belangrijk_bericht}
+          message={replaceVariablesInText(
+            text.varianten_tabel.belangrijk_bericht,
+            { sample_size: data.sample_size }
+          )}
           variant="emphasis"
         />
       )}
 
       <Box overflow="auto" mb={3} mt={4}>
-        {breakpoints.sm ? (
-          <DesktopVariantsTable rows={variantsTableRows} text={text} />
-        ) : (
-          <MobileVariantsTable rows={variantsTableRows} text={text} />
-        )}
+        <ErrorBoundary>
+          {breakpoints.sm ? (
+            <DesktopVariantsTable rows={variantsTableRows} text={text} />
+          ) : (
+            <MobileVariantsTable rows={variantsTableRows} text={text} />
+          )}
```

Comment:
nit; It's not really a desktop/mobile table because we detect a viewport width instead of a device.

```suggestion
          {breakpoints.sm ? (
            <WideVariantsTable rows={variantsTableRows} text={text} />
          ) : (
            <NarrowVariantsTable rows={variantsTableRows} text={text} />
          )}
```